bolt_unix.go

```diff
@@ -45,10 +45,10 @@ func funlock(f *os.File) error {
 }
 
 // mmap memory maps a DB's data file.
-func mmap(db *DB, sz int) error {
+func mmap(db *DB, sz int, changesize bool) error {
```

Comment:
Can you change the variable name to `syncRequired`? I think it makes more sense within the context of the function.